coders/heic.c

```diff
@@ -330,14 +335,12 @@ static MagickBooleanType ReadHEICImageByID(const ImageInfo *image_info,
   */
   (void) SetImageColorspace(image,YCbCrColorspace,exception);
   decode_options=(struct heif_decoding_options *) NULL;
-  option=GetImageOption(image_info,"heic:preserve-orientation");
-  if (IsStringTrue(option) == MagickTrue)
+  if (preserve_orientation == MagickTrue)
     {
       decode_options=heif_decoding_options_alloc();
       decode_options->ignore_transformations=1;
     }
   else
```

Comment:
Looks like this else statement should be gone?

Reply:
oeps :-O